src/Humanizer/CollectionHumanizeExtensions.cs

```diff
@@ -15,46 +15,46 @@ public static class CollectionHumanizeExtensions
         /// using the default separator for the current culture.
         /// </summary>
         /// <returns></returns>
-        public static string Humanize<T>(this IEnumerable<T> collection)
+        public static string Humanize<T>(this IEnumerable<T> collection, StringJoinOptions options = StringJoinOptions.Default)
```

Comment:
@aloisdg shorter: if `PublicApiApprovalTest.approve_public_api.approved.txt` has removed lines - it is a breaking change and we are forced to increase a major version part.